Dockerfile

```diff
@@ -1,14 +1,16 @@
-FROM ubuntu:16.04
+FROM ubuntu:16.10
```

Comment:
Good suggestion. Thanks! (I don't use any of this technology usually, so I'm happy about feedback from more experienced people!)